docs/MOS-API.md

```diff
@@ -889,7 +889,7 @@ The following system variables are available in [mos_api.inc](#usage-from-z80-as
 ; System variable indexes for api_sysvars
 ; Index into _sysvars in globals.asm
 ;
-sysvar_time:			EQU	00h	; 4: Clock timer in centiseconds (incremented by 2 every VBLANK)
+sysvar_time:			EQU	00h	; 4: Clock timer in centi seconds (incremented by 2 every VBLANK)
```

Comment:
I think this one was correct

"centi" as a word by itself isn't used in English.  instead it's used as a prefix to another word, without any hyphenation.

for example, we don't use "centi metre", or "centi gram"

that "centisecond" tends not to appear in most dictionaries is most likely just a result of it being an under-used term.

Reply:
```suggestion
sysvar_time:			EQU	00h	; 4: Clock timer in centiseconds (incremented by 2 every VBLANK)
```

Reply:
You are correct here (my bad) - when I first read it, it just looked wrong. I'm laughing now because there are lots of examples where it does make sense (centimenters, milimeters, megaton). I don't know what I was thinking.